app/assets/javascripts/discourse/app/components/glimmer-site-header.gjs

```diff
@@ -17,7 +16,7 @@ import { isDocumentRTL } from "discourse/lib/text-direction";
 import swipe from "discourse/modifiers/swipe";
 import { isTesting } from "discourse-common/config/environment";
 import discourseLater from "discourse-common/lib/later";
-import { bind, debounce } from "discourse-common/utils/decorators";
```

Comment:
PR should be merged after https://github.com/discourse/discourse/pull/29398, this line will likely result in a merge conflict.